README.md

```diff
@@ -20,7 +20,7 @@ You can install this chart using [`nri-bundle`](https://github.com/newrelic/helm
 
 ```shell
 helm repo add nri-prometheus https://newrelic.github.io/nri-prometheus
```

Comment:
The repository that we are adding for Prometheus is `nri-prometheus` so the install should be `nri-prometheus/nri-prometheus`.

Am I missing something?